awx/api/serializers.py

```diff
@@ -4563,13 +4573,34 @@ def validate(self, attrs):
                     not_allowed = requested_use_credentials - accessible_use_credentials
                     raise serializers.ValidationError(_(f"Credentials {not_allowed} not found."))
 
+            if requested_use_labels:
+                accessible_use_labels = {tup[0] for tup in Label.objects.all()}
+                if requested_use_labels - accessible_use_labels:
+                    not_allowed = requested_use_labels - accessible_use_labels
+                    raise serializers.ValidationError(_(f"Labels {not_allowed} not found"))
+
+            if requested_use_instance_groups:
+                # only org admins are allowed to see instance groups
```

Comment:
That is interesting. What about job template admins, can they not set IGs when they create a new JT?

Reply:
Yup only org admins. Not even workflow admin or job template admin.